Stretchy-type: Fix front-end view

tommusrhodus opened the pull request:
Updated plugin version to 0.1.1 in PHP, package, and block files. Changed update URLs and filter names to use opsoasis.mystagingwebsite.com. Removed unused 'render' property from block.json and improved code consistency in JS files.

Also addresses #76 by fixing the blockProps saved in save.js

Testing:
- First use 0.1.0 of the block, add to a page, add text and save.
- Observe the front-end does not resize the block.
- Load this patch, `npm build` the block locally, then re-save the page with the block in it.
- Observe that the expected stretch behaviour now takes place.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Added an "Update URI" field to the plugin metadata for improved update management.

* **Bug Fixes**
  * Updated plugin self-update mechanism to use the new domain for update checks.
  * Fixed an issue where a front-end CSS class was not being applied.

* **Documentation**
  * Removed the empty "example" property from the block manifest.

* **Style**
  * Standardized code formatting and quote usage for improved readability.

* **Chores**
  * Incremented version numbers to 0.1.2 in relevant files.
  * Updated block manifest to remove unused "render" property.

* **Enhancements**
  * Ensured consistent presence of SVG `viewBox` property and added a fixed CSS class for styling.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

tommusrhodus committed:
Bump version to 0.1.1 and update plugin URLs

Updated plugin version to 0.1.1 in PHP, package, and block files. Changed update URLs and filter names to use opsoasis.mystagingwebsite.com. Removed unused 'render' property from block.json and improved code consistency in JS files.

Also addresses #76 by fixing the blockProps saved in save.js

nate-allen commented on stretchy-type/classes/class-wpsp-blocks-self-update.php:
```diff
@@ -31,7 +31,7 @@ public static function get_instance() {
 	 */
 	public function hooks() {
-		add_filter( 'update_plugins_opsoasis-develop.mystagingwebsite.com', array( $this, 'self_update' ), 10, 3 );
+		add_filter( 'update_plugins_opsoasis.mystagingwebsite.com', array( $this, 'self_update' ), 10, 3 );
```
Should this be:

update_plugins_opsoasis.wpspecialprojects.com

tommusrhodus replied:
Whoops! Good catch, resolved :)

nate-allen commented on stretchy-type/classes/class-wpsp-blocks-self-update.php:
```diff
@@ -53,7 +53,7 @@ public function self_update( $update, array $plugin_data, string $plugin_file )
 		// Ask opsoasis.mystagingwebsite.com if there's an update.
 		$response = wp_remote_get(
-			'https://opsoasis-develop.mystagingwebsite.com/wp-json/opsoasis-blocks-version-manager/v1/update-check',
+			'https://opsoasis.mystagingwebsite.com/wp-json/opsoasis-blocks-version-manager/v1/update-check',
```
And this one should be:

https://opsoasis.wpspecialprojects.com/wp-json/opsoasis-blocks-version-manager/v1/update-check

tommusrhodus replied:
Whoops! Good catch, resolved :)

tommusrhodus committed:
Update plugin URLs to wpspecialprojects.com

Changed all references from opsoasis.mystagingwebsite.com to opsoasis.wpspecialprojects.com in self-update logic and plugin header to reflect new update server domain.

tommusrhodus committed:
Add className to SVG block props in save function

Added the 'wp-block-wpsp-stretchy-type' className to the SVG element's blockProps in the save function to ensure consistent styling and block identification.

tommusrhodus committed:
Merge branch 'trunk' into fix/76-stretchy-text-block-not-resizing

tommusrhodus committed:
Stretchy Type: Version bump
